SPECS/qemu/qemu.spec

```diff
@@ -2221,6 +2277,11 @@ useradd -r -u 107 -g qemu -G kvm -d / -s %{_sbindir}/nologin \
 
 
 %changelog
+* Wed Apr 20 2022 Daniel McIlvaney <damcilva@microsoft.com> - 6.2.0-1
+- Updated to match Fedora 36 (license: MIT)
+- Patched CVE-2022-0358
+
```

Comment:
Nit:

```suggestion
```

Reply:
Oops, just forgot to finish the changelog there.